beets/autotag/mb.py

```diff
@@ -217,6 +217,10 @@ def album_info(release):
         format = medium.get('format')
 
         all_tracks = medium['track-list']
+        if 'data-track-list' in medium:
+            all_tracks += medium['data-track-list']
+        total = len(all_tracks)
```

Comment:
I had elected to name this variable `total` because the track index in the same function is named `index`. Likewise, the corresponding parameter of `track_info` is `medium_total`. Sounded kind of consistent to me. But I don't mind changing its name to anything else which you'd think is more explicit :)

Cheers,
Jérémie.